projects/kit/components/combo-box/combo-box.component.ts

```diff
@@ -139,15 +139,18 @@ export class TuiComboBoxComponent<T>
     }
 
     get nativeValue(): string {
-        return this.value === null ? this.search || `` : this.stringify(this.value);
+        return this.value === null ? this.search ?? `` : this.stringify(this.value);
     }
 
     get showValueTemplate(): boolean {
         return tuiIsPresent(this.value) && !this.focused;
     }
 
     get computedContent(): PolymorpheusContent<TuiValueContentContext<T>> {
-        return this.valueContent || this.nativeValue;
+        return (
+            (this.valueContent as Exclude<PolymorpheusContent, null | undefined>) ||
```

Comment:
I don't like these typecasts. Like I said before, maybe move it to warning just to pay more attention to possibility of an issue with `||` and `??`.